README.md

```diff
@@ -18,7 +18,7 @@ Developed against Python 3.9.1.
 1. Install required modules: `pip install -r requirements.txt`
```

Comment:
I would include a loud speaker at the top announcing the OCS to ADH change.  Something like:

| :loudspeaker: **Notice**: Samples have been updated to reflect that they work on AVEVA Data Hub.  The samples also work on OSIsoft Cloud Services unless otherwise noted. |
| -----------------------------------------------------------------------------------------------|

Reply:
Sorry I thought we wanted just one on the OSI-Samples-OCS repo level, I'll add these here on the sample level instead. Thank you!